diff --git a/stage4/06-enable-wayland/00-run.sh b/stage4/06-enable-wayland/00-run.sh
index d1f240d..d494c61 100755
--- a/stage4/06-enable-wayland/00-run.sh
+++ b/stage4/06-enable-wayland/00-run.sh
@@ -1,5 +1,5 @@
 #!/bin/bash -e

-on_chroot << EOF
-	SUDO_USER="${FIRST_USER_NAME}" raspi-config nonint do_wayland W3
+on_chroot <<EOF
+	SUDO_USER="${FIRST_USER_NAME}" raspi-config nonint do_wayland W1
 EOF
