diff --git a/stage4/00-install-packages/00-packages b/stage4/00-install-packages/00-packages
index 511941c..238f0f7 100644
--- a/stage4/00-install-packages/00-packages
+++ b/stage4/00-install-packages/00-packages
@@ -1,17 +1,13 @@
-python3-pygame
-python3-tk thonny
+python3-tk
 python3-pgzero
 python3-serial
-debian-reference-en dillo
 raspberrypi-net-mods raspberrypi-ui-mods
 python3-pip
 python3-numpy
-alacarte rc-gui sense-hat
+alacarte rc-gui
 tree
 libgl1-mesa-dri libgles1 libgles2-mesa xcompmgr
-geany
 piclone
-python3-twython
 python3-flask
 pprompt
 piwiz
